src/pkg/podman/podman.go

```diff
@@ -320,15 +338,31 @@ func Start(container string, stderr io.Writer) error {
 }
 
 func SystemMigrate(ociRuntimeRequired string) error {
+	stderr := setupStderrWriter(nil)
+
 	logLevelString := LogLevel.String()
 	args := []string{"--log-level", logLevelString, "system", "migrate"}
 	if ociRuntimeRequired != "" {
 		args = append(args, []string{"--new-runtime", ociRuntimeRequired}...)
 	}
 
-	if err := shell.Run("podman", nil, nil, nil, args...); err != nil {
+	if err := shell.Run("podman", nil, nil, stderr, args...); err != nil {
 		return err
 	}
 
 	return nil
 }
+
+func setupStderrWriter(buffer io.Writer) io.Writer {
```

Comment:
It will be better to put this inside `pkg/shell`, because we want the same behaviour for all spawned binaries. As a nice side-effect, it will make the change more self-contained and smaller.